docs/02-for-app-authors/03-appdata-guidelines/01-quality-guidelines.md

```diff
@@ -0,0 +1,69 @@
+# Quality guidelines
+
+For the quick guidelines for the actual content (descriptions, screenshots), see the Appstream [docs](https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html).
+
+The following guidelines are optional, but recommended. We will likely promote apps that follow them in certain places.
+
+## General
+
+### No trademark violations
+
+Your app must not violate any trademarks. This includes the name, icon, and summary.
+
+### Managed by app author
+
+We recommend to verify your app on Flathub, but it’s not required.
+
+## App Icon
+
+### Icon size
+
+The icon must be at least 256x256px and be in PNG or SVG format.
+
+### Icon ratio
+
+The icon shouldn't fill less than 90% of the canvas or touch the edges.
+
+### Good contrast
+
+The icon should have good contrast on dark and light backgrounds.
```

Comment:
```suggestion
The icon should have good contrast on dark and light backgrounds. In particular, avoid single-color glyphs and the like since they can be very hard to see if the background color is similar. Dark colors near the edges are also a problem, since shadows don't help much with contrast on dark backgrounds.
```